spec/features/course/category_management_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.feature 'Course: Category: Management' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    before { login_as(user, scope: :user) }
+
+    context 'As a Course Manager' do
+      let(:user) { course.creator }
+      scenario 'I can create a new category' do
+        category = build_stubbed(:course_assessment_category)
+
+        visit course_admin_assessments_path(course)
+        find_link(nil, href: new_course_admin_category_path(course)).click
+
+        expect(current_path).to eq(new_course_admin_category_path(course))
+        fill_in 'title', with: category.title
+        fill_in 'weight', with: category.weight
+        click_button 'submit'
+
+        expect(current_path).to eq(course_admin_assessments_path(course))
+        expect(page).to have_selector('div.alert.alert-success')
+      end
+
+      scenario 'I can edit category fields' do
+        category = build_stubbed(:course_assessment_category)
+        category_edited = build_stubbed(:course_assessment_category)
+
+        visit course_admin_assessments_path(course)
+
+        find_link(nil, href: new_course_admin_category_path(course)).click
+        fill_in 'title', with: category.title
+        fill_in 'weight', with: category.weight
+        click_button 'submit'
+
+        category_input = '#course_assessment_categories_attributes_0_%s'
```

Comment:
This was intended to test the ordering. Would a model test be more suitable then?